### DISCUSSION TITLE
Nice work - a request

### DISCUSSION THREAD

**Discussion by Xoliul** [danielrivard/homeassistant-innova#50] [Ideas]
Very happy to find this and have it working!
Some thoughts:
The "Aircon" entitity it adds is quite monolithic, adding it to the Lovelace UI isn't ideal. I'd love to be able to just get the temperature sensor from the unit as a separate entity. Would it be possible to add that?

**Comment by danielrivard** [comment]
I was actually thinking of adding an ambient temperature sensor. I just did not have the time to put my mind into it yet. But I should be able to find some this fall.

**Comment by danielrivard** [comment]
Hi @Xoliul, I have just released a new version with an ambient temperature sensor entity.
